lib/v2/xmnn/news.js

```diff
@@ -0,0 +1,69 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const { category = 'xmxw' } = ctx.params;
+    const limit = ctx.query.limit ? parseInt(ctx.query.limit, 10) : 30;
+
+    const rootUrl = 'https://news.xmnn.cn';
+    const currentUrl = new URL(category, rootUrl).href;
+
+    const { data: response } = await got(currentUrl);
+
+    const $ = cheerio.load(response);
+
+    let items = $('div#sort_body ul li a')
+        .slice(0, limit)
+        .toArray()
+        .map((item) => {
+            item = $(item);
+
+            return {
+                title: item.find('h1').text().trim(),
+                link: item.prop('href'),
+                description: item.find('div.abstract').html(),
+                author: item.find('div.source').text(),
+                category: item
+                    .find('.category')
+                    .toArray()
+                    .map((c) => $(c).text()),
```

Comment:
`category` is empty for all 9 categories listed in the docs

Reply:
👌